docs/services/data/configure-data-query.md

```diff
@@ -0,0 +1,93 @@
+---
+title: "Configure Data Query"
+linkTitle: "Configure Data Query"
+description: "Configure data query to query synced data directly in the cloud."
+weight: 35
+type: "docs"
+tags: ["data management", "cloud", "query"]
+# SME: Aaron Casas
+---
+
+Configure data query to be able to directly query captured data in the Viam cloud using the [MongoDB Query Language (MQL)](https://www.mongodb.com/docs/manual/tutorial/query-documents/).
+
+## Requirements
+
+Before you can configure [data query](../#data-query), you must [add the data management service](/services/data/configure-data-capture/#add-the-data-management-service) and [configure cloud sync](/services/data/configure-cloud-sync/).
+
+## Configure data query
+
+Once your smart machine has synced captured data to the Viam app, you can configure data query using the Viam CLI:
+
+1. If you haven't already, [install the Viam CLI](/manage/cli/#install) and [authenticate](/manage/cli/#authenticate) to Viam.
+
+1. Find your organization ID by running the following command, or from your organization's **Settings** page in [the Viam App](https://app.viam.com/):
```

Comment:
[nit] are these numbers from here onwards supposed to increment, or does it render differently?